lib/index.js

```diff
@@ -31,32 +30,6 @@ function getResultSeverity(resultMessages = []) {
   }, 0);
 }
 
-/**
```

Comment:
This whole filtering logic is superseded by the new guard which keeps ignored files from being linted on at all. So it is no longer necessary. I kept the tests in to ensure however.